types/properties-reader/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for properties-reader 0.0
+// Project: https://github.com/steveukx/properties
+// Definitions by: Zlatko Andonovski <https://github.com/Goldsmith42/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace PropertiesReader {
+    type Value = string|number|boolean;
+
+    interface Reader {
+        get(propertyName: string): Value|null;
+        getRaw(propertyName: string): string|null;
+        path(): Object;
```

Comment:
I did some testing with this, it does offer some extra safety if you're trying to do something silly like cast the result to `Function`.